fuzzers/fuzzer.ts

```diff
@@ -23,6 +54,20 @@ export class FuzzCase {
 		this.contextItem = contextItem;
 	}
 
+	/**
+	 * Tests whether the given error is expected,
+	 * meaning the error should not be considered a crash.
+	 * 
+	 * @param error - The {@link Error} which to test.
+	 * @returns Returns `true` when expected.
+	 */
+	isExpectedError(error: Error): boolean {
+		return EXPECTED_ERROR_PREFIXES.some(prefix => error.message.startsWith(prefix));
```

Comment:
Maybe use a `Set`. It looks like all error messages have the same length, so something like `EXPECTED_ERROR_PREFIXED.has("ABCD000".slice(0, 4))` would work. Could give a small performance boost.

Reply:
Good point `O(1)` vs `O(n)` where `n` is the length of `EXPECTED_ERROR_PREFIXED`. Given the rest of the code, I doubt it will make an big impact on performance though. It should be measurable in a benchmark.